### ISSUE TITLE
Input other than enum?

### ISSUE THREAD

**Issue by eitsupi** [devcontainers/cli#369]

**Comment by eitsupi** [comment]
Ah, I obviously posted in the wrong repository! (I was going to post this on the feature-starter repo)
But I glad you now realize that you have the same case on the features repo.

As you say, it would be great so that in the future such cases can raise errors.

**Comment by samruddhikhandale** [comment]
> Ah, I obviously posted in the wrong repository! (I was going to post this on the feature-starter repo)

You did post in feature-starter repo, I transferred to the CLI repo. I think the change should be made to the cli. @eitsupi  Do you have other thoughts?

**Comment by eitsupi** [comment]
> You did post in feature-starter repo, I transferred to the CLI repo. I think the change should be made to the cli.

👍

> Do you have other thoughts?

Right now we need to read the json file to know what values are allowed for the enum type, it would be great if we could get them to show up in the README.

**Comment by samruddhikhandale** [comment]
> Right now we need to read the json file to know what values are allowed for the enum type, it would be great if we could get them to show up in the README.

Currently, docker-in-docker Feature uses an enum for an option.


![image](https://user-images.githubusercontent.com/24955913/211120202-a10c22d8-722d-42ed-9797-3af98eee9741.png)

https://github.com/devcontainers/features/blob/main/src/docker-in-docker/devcontainer-feature.json#L23-L30

The description mentions the accepted values in bracket (in this case v1/v2). Would updating description sufficient? 

We could update devcontainers/action to populate this in the REAMDE but I think the use of `enum` is comparatively less. Not sure if the `options` section might get crowded 🤔 

![image](https://user-images.githubusercontent.com/24955913/211120340-20b0ee3c-e334-4b88-b259-5d40eb8a7c6d.png)

**Comment by eitsupi** [comment]
Yes, in any case, the choices must be explained in the description.
Nevertheless, it would be useful to add a second table with details of the enum type options, I think.

For example, in the case of an option like version, the meaning of the choices would be self-explanatory, so there would be no need to explain everything in the description.
https://github.com/rocker-org/devcontainer-features/blob/5f2bb3db326f41b334a588a9116121e8a4409ede/src/r-rig/devcontainer-feature.json#L18-L27
